projects/angular-material-extensions/select-country/src/lib/mat-select-country.component.ts

```diff
@@ -67,6 +68,9 @@ export class MatSelectCountryComponent implements OnInit, OnChanges, ControlValu
   }
 
   ngOnInit() {
+    if(this.customCountryList){
+      this.countries = this.customCountryList;
```

Comment:
I think this is overlapping the `countries` property.


I think the best way to use `countries` as Input to predefine the countries options for the end user